<launch>
	<include file="$(find-pkg-share openni2_camera)/launch/camera_only.launch.py"/>
  <node pkg="rgbd_slam" 		exec="depth_image_odometry_node" name="depth_image_odometry_node" output="screen">
		<param name="rgb_topic" 	     				value="/camera/rgb/image_raw" />
		<param name="depth_topic" 	     			value="/camera/depth/image_raw" />
		<param name="calibration_file" 	 			value="$(find-pkg-share rgbd_calibration)/config/rgbd_calibration.txt" />

		<!--Feature detection-->
		<param name="im_width" 						 								value="640" />
		<param name="im_height" 					 								value="480" />
	</node>
</launch>